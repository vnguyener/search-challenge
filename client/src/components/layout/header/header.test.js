import React from 'react';
import renderer from 'react-test-renderer';

import Header from '.';

describe('layour header component', () => {
  it('renders correctly', () => {
    const element = renderer.create(<Header />).toJSON();
    expect(element).toMatchSnapshot();
  });
});
