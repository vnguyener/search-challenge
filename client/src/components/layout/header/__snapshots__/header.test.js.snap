// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`layour header component renders correctly 1`] = `
<header
  className="main-header"
>
  <img
    alt="match"
    src="./logo.svg"
    width="110"
  />
</header>
`;
